src/locales/en/translation.json

```diff
@@ -813,6 +819,34 @@
         "line_2": "Currently pool has only {{amount}} {{currency}} available.",
         "line_3": "Pool will increase once users will repay their loans."
       }
+    },
+    "poolChart": {
+      "totalLiquidity": "Total Liquidity",
+      "apy": "{{asset}} APY%"
+    },
+    "rowTable": {
+      "noLiquidityProvided": "Lend your asset and earn income",
```

Comment:
In my opinion, the (asset) word should be replaced by the asset name. rBTC, DoC....